lib/liquid.rb

```diff
@@ -35,7 +35,8 @@ module Liquid
   QuotedFragment              = /#{QuotedString}|(?:[^\s,\|'"]|#{QuotedString})+/o
   TagAttributes               = /(\w+)\s*\:\s*(#{QuotedFragment})/o
   AnyStartingTag              = /\{\{|\{\%/
-  PartialTemplateParser       = /#{TagStart}.*?#{TagEnd}|#{VariableStart}.*?#{VariableIncompleteEnd}/om
+  tag_contents                = /(?:#{QuotedString}|.)*?/m
```

Comment:
also, the local variable is because I don't want it to be used outside of liquid